member_organizations.yaml

```diff
@@ -28,6 +28,15 @@
   hiringForKubeflow: true
   hiringUrl: https://caicloud.io/jobs
   
+- name: Heptio
```

Comment:
nit; could you please make this alphabetical by company name?

Reply:
Sure - one second